turbinia/celery.py

```diff
@@ -142,6 +142,11 @@ def check_messages(self):
         break
       except ChannelError:
         break
+      except Exception as e:
```

Comment:
I'm always pretty hesitant to trap exceptions for something so broad (though admittedly there is one other place in the codebase that does this).  Is there any way we can scope this a bit tighter?  Have you seen general problems that would require this?  I see that kombu does have a KombuError exception class that the other exceptions use as a base class.  Maybe we could add that here instead?
http://docs.celeryproject.org/projects/kombu/en/latest/_modules/kombu/exceptions.html

Reply:
Good call. The initial problem I can into is that each transport can throw their own custom exceptions here.

I did some more digging and there is actually a property/list `recoverable_channel_errors` which can be populated by the transport. `OperationalError` might even give us what we need. I will try to test.

Reply:
Ah, I see.  It would be nice if Kombu caught those and re-raised its own exceptions.  It might be worth a feature request to them.

LMK when this is ready for re-review.

Reply:
Catching Kombu's `OperationalError` now.